shopper_ds/shopper.py

```diff
@@ -57,6 +57,15 @@ def safe_init_device(self):
         description="Attribute that represents PLC signal for shutter in "
                     "interlock alarm.")
 
+    # state attributes
+
+    @state_attribute(
+        bind=['ShopperInterlock'])
+    def InterlockAlarm(self, alarm):
```

Comment:
Since it's a normal method and not a part of Tango interface, this method should be named using `snake_case`.